### PR TITLE
Add documentation to data.py

### PR DESCRIPTION
**Pull request by orkosinha**
Address issue #66

### CODE REVIEW

**Review by ilikesymmetry — APPROVED** [review]
so fresh and so cleeeaaaan

**Review by young-k — CHANGES_REQUESTED** [review]
Can these messages be reformatted according to: https://google.github.io/styleguide/pyguide.html#383-functions-and-methods

```
def example():
    """ This should be a one sentence summary.

    Longer summaries that span multiple lines 
    should go here

    Args:
         ...
"""
```

**Review by young-k — COMMENTED** [review]
The branch name should also be `orko/documentation`, you can rename a branch with `git branch -m [BRANCH_NAME]`, but that's for next time!

File: src/data.py [hunk 1]
```diff
@@ -93,12 +112,19 @@ def parse_payment_methods(methods):
...
+      dining_items (list): A list that holds a dictionary for the items an eatery serves and a flag
+       for healthy options
+  """
   new_operating_hours = []
   hours_list = eatery['operatingHours']
   for hours in hours_list:
     new_date = hours.get('date', '')
     hours_events = hours['events']
-
+    #merge the dining items from the json (for non-dining hall eateries) with the menu for an event
```
Comment (meganle) [inline src/data.py:137]: Let's add a space between the '#' and 'merge'!

**Review by meganle — APPROVED** [review]
Everything looks good, great job! Just address Young's comment about the formatting and we can merge

File: src/data.py (hunk 1 above)
Comment (orkosinha) [reply src/data.py]: Added it back!

File: src/data.py [hunk 2]
```diff
@@ -93,12 +120,22 @@ def parse_payment_methods(methods):
...
+      dining_items (list): A list that holds a dictionary for the items an eatery serves and a flag
+       for healthy options
+  """
   new_operating_hours = []
   hours_list = eatery['operatingHours']
   for hours in hours_list:
     new_date = hours.get('date', '')
     hours_events = hours['events']
 
+    #merge the dining items from the json (for non-dining hall eateries) with the menu for an event
```
Comment (young-k) [inline src/data.py:137]: this space is still missing lol

**Review by young-k — APPROVED** [review]
LGTM, can you squash the two commits into one?

File: src/data.py (hunk 2 above)
Comment (orkosinha) [reply src/data.py]: oops

Comment (orkosinha) [reply src/data.py:137]: squashed and resolved!

**Pull request merged**